source/code-generation/query-converter/convert-stored-procedures.txt

```diff
@@ -74,9 +74,9 @@ Steps
 
          .. tip::
 
-            You can toggle stored procedures for conversion by clicking
-            the :icon-fa5:`check-square` icon next to a stored procedure's
-            name. All stored procedures are selected by default.
+            Click the :icon-fa5:`check-square` icon next to a stored 
+            procedure's name to toggle it for conversion. All stored 
```

Comment:
s/b something like:
To toggle stored procedures for conversion, click the check-square icon next to the stored procedure's name. 

Also, question. Maybe it's a little more obvious in context, but what does "toggle stored procedures for conversion" mean?

Reply:
Toggling the stored procedure with the tick box means it will be in context for the query converter to convert.